addons/mail/static/src/core/common/chat_bubble.js

```diff
@@ -33,10 +33,10 @@ export class ChatBubble extends Component {
         this.rootRef = useRef("root");
         this.state = useState({ bouncing: false, showClose: true });
         useEffect(
-            () => {
-                this.state.bouncing = this.thread.importantCounter ? true : this.state.bouncing;
+            (importantCounter) => {
+                this.state.bouncing = Boolean(importantCounter);
```

Comment:
Is it really the same?

When the value was falsy it used to keep the original state of `this.state.bouncing`, not reset it to false.

Reply:
Isn't the state bouncing useless then? It's just the value of `this.thread.importantCounter` at any time, except it is done post rendering with a useEffect, which might have the side effect to create more renders than necessary?

Reply:
Before:
![1](https://github.com/user-attachments/assets/73ea3b12-6c6b-4d77-8b0a-bf1b7c3ab27e)
After:
![2](https://github.com/user-attachments/assets/714eb257-d543-4a8e-a934-bc03949c80fd)